Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
Package unit tests should be run by the upstream project. If and when they catch Homebrew bugs, we should run them. They don't seem to, though. `test do` catches Homebrew or upstream issues if the software doesn't work as expected.